@here/harp-mapview/lib/MapView.ts

```diff
@@ -997,6 +998,14 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_textElementsRenderer;
     }
 
+    /**
+     * @hidden
+     * Is `true` if the TextElementRenderer is soon to be created.
+     */
+    get textElementsRendererRequested(): boolean {
```

Comment:
Why exposing so obscure internal property esp, it's only used once.

Reply:
TextElementsRenderer is public, there is no control over when/if it is created. Now it is accessible if it _will_ be available soon. Given the fact that we have/had so many issues with that first frame complete event, I think it is valuable.

Reply:
Can we instead do it properly and always create the TextElementsRenderer?
I understand that we don't want to create it if we have no text b/c we would load the font for nothing, but I think we should load the font lazily instead of creating the whole TextElementsRenderer lazily

Reply:
This will increase startup time. @Nino: Please, we need executive decision on that.

Reply:
Why will this increase startup time? As I said, the text elements renderer should always be created but the font's inside the text elements renderer should be loaded lazily. Please sync with @atomicsulfate He is refactorin TextElementsRenderer right now.